appbase/publishers.py

```diff
@@ -125,6 +125,9 @@ def cached(f):
     if hasattr(f, 'cache'):
         cf = cache(f)
         cf.began = datetime.datetime.now()
+        # checks if the function has 'cache_ttl' attribute which stores a specific cache expiry time for it,
+        # else uses the default cache_ttl
+        cache_ttl = getattr(f, 'cache_ttl', cache_ttl)
```

Comment:
@pramitrn I think it will override default `cache_ttl`
we need to rename default `cache_ttl` to `default_cache_ttl`
cc: @shon

Reply:
@gauravr right and thanks. Better to use `default_cache_ttl` so that it doesn't accidentally get overridden
Having said that, I am not to sure if this code will 
- override cache_ttl as it's not marked global in the function
- most likely that code might fail to find ttl in the namespace

```
ttl = 10

def test(i=None):
    if i and (i % 2):
        print(f'i = {i}')
        ttl = 99
    print(f'ttl = {ttl}')

test()
for i in range(10):
    test(i)
```